projects/risk-assessment-scores-to-delius/README.md

```diff
@@ -1,8 +1,47 @@
 # Risk Assessment Scores to Delius
 
-This integration listens for domain events generated by the OASys Offender Assessment System, and stores changes to the
-RSR (Risk of Serious Recidivism) and OSP (OASys Sexual reoffending Predictor) scores in the Delius probation case
-management system.
+This integration service listens for domain events generated by the OASys Offender Assessment system and updates Delius with changes to the following risk scores:
+
+* Risk of serious recidivism (RSR) and OASys sexual reoffending predictor (OSP) scores
+* Offender group reconviction scale (OGRS) scores, which are updated when a OGRS risk assessment is submitted for a **Person on Probation**.
```

Comment:
Sorry if I'm mis-reading this, is it trying to indicate that only the OGRS assessment is for a person on probation?  All of the RSR, OSP and OGRS scores are stored against the person on probation's case - it's just that the RSR and OSP ones are added by a stored procedure in the database, whereas we create the records manually for OGRS.